sbc_compassion/models/correspondence.py

```diff
@@ -698,7 +698,11 @@ def download_attach_letter_image(self, type='final_letter_url'):
                 raise UserError(
                     _("Image of letter {} was not found remotely.").format(
                         letter.kit_identifier))
-            name = letter.child_id.local_id + '_' + letter.kit_identifier + \
+            name = ''
+            if len(letter.communication_type_ids.ids) > 0:
+                name = letter.communication_type_ids[0].name + '_'
```

Comment:
you can do `letter.communication_type_ids[0].with_context(lang=letter.correspondant_id.lang).name` in order to translate the type in the language of the sponsor. Add as well the attribute `translate=True` in the field declaration in `models/correspondence_template.py` at line 60.

Reply:
Change the `_` by a space